receiver/sqlqueryreceiver/README.md

```diff
@@ -24,6 +24,7 @@ The configuration supports the following top-level fields:
   e.g. _host=localhost port=5432 user=me password=s3cr3t sslmode=disable_
 - `queries`(required): A list of queries, where a query is a sql statement and one or more `logs` and/or `metrics` sections (details below).
 - `collection_interval`(optional): The time interval between query executions. Defaults to _10s_.
+- `storage` (optional, default `""`): The ID of a storage extension to be used to [track processed results](#tracking-processed-results).
```

Comment:
please add here link to the documentation of storage extension

Reply:
Added link to the File Storage extension. I couldn't find general documentation about storage extensions.